### PR TITLE
Update tools chapter

### PR DESCRIPTION
**Pull request by nives113**

### CODE REVIEW

File: Code Quality/Tools.md [hunk 1]
```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
...
-```js
-// package.json
-{
-  "husky": {
-    "hooks": {
-      "pre-push": "npm test"
-    }
-  }
-}
+Follow [usage](npm set-script prepare "husky install") guidelines for installing Husky and add hooks.
```
Comment (fvoska) [inline Code Quality/Tools.md:11]: I think you wanted to paste some URL to husky docs here, not some random command :D

File: Code Quality/Tools.md [hunk 2]
```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
...
+
+After running all commands described [usage](npm set-script prepare "husky install") chapter, you should have `.husky` folder with hooks folder inside. For example, if you created `pre-commit` hook which runs `npm test` command, you should have `pre-commit` file in `.husky` folder, with following content
```
Comment (fvoska) [inline Code Quality/Tools.md:13]: Same comment about link

Comment (fvoska) [reply Code Quality/Tools.md:13]: `After running all commands described in [usage](...) chapter`

File: Code Quality/Tools.md [hunk 3]
```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
...
+
+```sh
+# .husky/pre-commit
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+cd frontend && npx lint-staged
+
+```
+
+For more use cases please check [Husky documentation](https://typicode.github.io/husky/#/).
```
Comment (fvoska) [inline Code Quality/Tools.md:36]: You could probably link directly to the relevant chapter in the docs: https://typicode.github.io/husky/#/?id=custom-directory

File: Code Quality/Tools.md [hunk 4]
```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
...
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "**/*.{js,ts}": [
+  "**/*.{js,ts}": [
       "eslint",
       "prettier --write"
```
Comment (fvoska) [inline Code Quality/Tools.md:53]: Move prettier above eslint

File: Code Quality/Tools.md [hunk 5]
```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
...
-    ]
-    "**/*.scss": [
+  ]
+  "**/*.scss": [
       "stylelint --syntax=scss"
```
Comment (fvoska) [inline Code Quality/Tools.md:56]: Add stylelint

File: Code Quality/Tools.md [hunk 6]
```diff
@@ -168,72 +186,90 @@ Infinum created various rulesets for ESLint and TSLint which you can check out [
...
 
-Here is the complete example which runs TypeScript compilation check on all files, prettier, tslint and stylelint on an Angular (v10) project:
+Here is the complete example which runs TypeScript compilation check on all files, prettier, eslint and stylelint on an Angular (v10) project:
 
 ```js
-// package.json
 {
   "scripts": {
-    "lint:ng": "ng lint",
-    "tsc": "concurrently \"npm run tsc:app\" \"npm run tsc:spec\"",
```
Comment (fvoska) [inline Code Quality/Tools.md:180]: tsc-related scripts should still be here and you should add `npm run tsc` to husky pre-commit hook (in the new format in .husky/pre-commit)

**Review by fvoska — CHANGES_REQUESTED** [review]

File: Code Quality/Tools.md (hunk 3 above)
Comment (nives113) [reply Code Quality/Tools.md:36]: Not really, I really wanted to set link for general documentation for all other possible use cases. 😄 
If you have any suggestions for rephrasing this part, please share

File: Code Quality/Tools.md (hunk 5 above)
Comment (nives113) [reply Code Quality/Tools.md:56]: You mean 'prettier'?

**Review by fvoska — APPROVED** [review]

**Pull request merged**